lib/diagnostic.rb

```diff
@@ -125,25 +151,25 @@ def eat_meat(food)
 
 # ComboAttack Class definition
 class ComboAttack
-  attr_reader :moves, :damage
+  attr_accessor :moves, :damage
 
   def initialize
     @moves = []
     @damage = 0
   end
 
-  def punch
-    @moves << 'punch'
+  def self.punch
```

Comment:
None of the existing methods need to be modified into class methods.  All of the public instance methods just need to return `self` to make the methods chainable.